docs/production-deployment/cloud/rto-rpo.mdx

```diff
@@ -19,75 +19,99 @@ tags:
 
 import { ToolTipTerm } from '@site/src/components';
 
-Recovery Point Objective (RPO) and Recovery Time Objective (RTO) define data durability and service restoration timelines, respectively.
-In Temporal Cloud, these objectives vary depending on your deployment configuration and the scope of any failure.
+Recovery Point Objective (RPO) and Recovery Time Objective (RTO) are the objectives that Temporal strives to meet for data durability (recovery point) and service restoration (recovery time) in the event of cloud outages.
+These objectives are high priority goals for Temporal Cloud, but are not a contractual commitment.
```

Comment:
This sounds rough. Can we say RPO + RTO aren't part of the availability SLA instead (and link to the SLA page)?

Reply:
Proposed: These objectives are complementary to but different from Temporal Cloud's [Service Level Agreement (SLA)](/cloud/sla).

Reply:
```suggestion
Temporal Cloud's RPO and RTO are complementary to but separate from the [availability SLA](/cloud/sla)."
```